apps/dashboard/src/app/components/DashboardTypeBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import { AnnouncementBanner } from "../../components/notices/AnnouncementBanner";
+
+export function TeamsUIBanner() {
+  return (
+    <AnnouncementBanner
+      label="Teams UI is still in Beta, Go to Old dashboard for fully functional dashboard"
```

Comment:
language should be less scary:

"You are exploring the new dashboard. Click here to go back to the legacy dashboard."

Reply:
updated